file_system.go

```diff
@@ -112,6 +112,27 @@ func (c *Client) Getwd() (string, error) {
 	return dir, nil
 }
 
+// Setwd changes the current working directory.
+func (c *Client) Setwd(path string) (error) {
```

Comment:
What if each connection remembers what dir it is in, and when a new connection is gotten it automatically does the cwd if it isn't in the correct dir (user's desired dir is stored on client object when they set via setwd())?